packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -3,23 +3,15 @@ namespace rootnamespace
 {
     using NServiceBus;
 
-    /*
-		This class configures this endpoint as a Server. More information about how to configure the NServiceBus host
-		can be found here: http://particular.net/articles/the-nservicebus-host
-	*/
     public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(BusConfiguration configuration)
         {
-            // NServiceBus provides the following durable storage options
-            // To use RavenDB, install-package NServiceBus.RavenDB and then use configuration.UsePersistence<RavenDBPersistence>();
-            // To use SQLServer, install-package NServiceBus.NHibernate and then use configuration.UsePersistence<NHibernatePersistence>();
+            // NServiceBus provides multiple durable storage options
```

Comment:
@SimonCropp - I had specifically added RavenDB and SqlServer. Telling the user we have multiple storage options but not tell them anything serves no purpose.

Instead, how about:
NServiceBus provides multiple durable storage options like RavenDB, SQLServer. Download the appropriate NuGet package and call the configuration.UsePersistence<T> API.